Docker/Dockerfile

```
@@ -63,7 +63,6 @@ ENV TRUSTED_PROXY=''
 ENTRYPOINT ["./Docker/entrypoint.sh"]
 
 EXPOSE 80
+
 # hadolint ignore=DL3025
-CMD ([ -z "$CRON_MIN" ] || cron) && \
-	. /etc/apache2/envvars && \
-	exec apache2 -D FOREGROUND $([ -n "$OIDC_ENABLED" ] && [ "$OIDC_ENABLED" -ne 0 ] && echo '-D OIDC_ENABLED')
+CMD sh -c '[ -z "$CRON_MIN" ] || cron; . /etc/apache2/envvars; if [ "${OIDC_ENABLED:-0}" -ne 0 ]; then exec apache2 -D FOREGROUND -D OIDC_ENABLED; else exec apache2 -D FOREGROUND; fi'
```

Nothing in that command is Bash syntax so I'd rather not rewrite the command without the && to fail early for example.

---

Or put another way, you could just change the SHELL command to `/bin/sh`.

---

Edits welcome

---

I wouldn't be inclined to edit anything at all.

You might phrase that as:
```suggestion
CMD sh -c '([ -z "$CRON_MIN" ] || cron) && . /etc/apache2/envvars && exec apache2 -D FOREGROUND $([ -n "$OIDC_ENABLED" ] && [ "$OIDC_ENABLED" -ne 0 ] && echo "-D OIDC_ENABLED")'
```

But in doing so I just realized something. Naturally I had to change `echo '-D OIDC_ENABLED'` to make that work, which might indicate they're not ignoring the SHELL command but are astoundingly bad at escaping, because it'd need to become either `\'` or `"` to work as part of `-c`.

```suggestion
CMD ([ -z "$CRON_MIN" ] || cron) && \
	. /etc/apache2/envvars && \
	exec apache2 -D FOREGROUND $([ -n "$OIDC_ENABLED" ] && [ "$OIDC_ENABLED" -ne 0 ] && echo "-D OIDC_ENABLED")
```

Because if you don't do that, the error you'll see is:

```
Syntax error: end of file unexpected (expecting ")")
```

Which I believe is the exact error in the referenced issue.

---

In fact on closer inspection, the specific form of the error `-c: line 2: unexpected EOF while looking for matching `)'` is how Bash presents that syntax error! The one I quoted above was from Dash.

---

What about now, with the `&&`s?

---

Ok, could be good to test with one of the failing systems

---

What about now?
https://github.com/FreshRSS/FreshRSS/pull/7861/commits/c8f3ec86b6181756a74240130fe8ae4640a2ed21

---

lgtm in any case